### ISSUE TITLE
Config File Parse Error

### ISSUE THREAD

**Issue by brian-js** [maidsafe/crust#313]
PR for routing running on Travis, https://travis-ci.org/maidsafe/routing/jobs/79325378, gave the following error,
...
895test public_id::test::assign_relocated_name_public_id ... ok
896ERROR:crust::file_handler: Failed to decode "/Users/travis/.config/routing-987caed5327961dd/routing-987caed5327961dd.crust.config": ParseError(SyntaxError("EOF While parsing object", 1, 2))

Appears the file can be opened and read but not decoded.

**Comment by Fraser999** [comment]
This looks like the sort of errors I saw in Crust when I ran the tests in parallel - i.e. without the env var `RUST_TEST_THREADS` set to `1`.

This makes sense since any exe linked to Crust will try and create a Crust config file in the "app support dir" if it can't find one in any of its search paths (see https://github.com/maidsafe/crust/blob/master/docs/vault_config_file_flowchart.pdf)

In this case, I'm guessing that one of the tests was still in the process of writing the file while the failing test found it, read it and couldn't parse it.

I think this should be closed as a Crust issue, and resubmitted as issues to all dependent libraries, but only after discussion with @dirvine.
